frontend/app/model/Zuora.scala

```diff
@@ -55,8 +55,10 @@ object Zuora {
   }
   case class InternalError(code: String, message: String) extends Error
 
-  case class SubscriptionStatus(current: String, future: Option[String], amendType: Option[String]) {
-    val cancelled = amendType.exists(_ == "Cancellation")
+  case class SubscriptionStatus(current: Subscription, future: Option[Subscription], amendType: Option[String]) {
```

Comment:
Also, the wording here can be a bit confusing, `future` suggests a Scala `Future`. Perhaps `newVersion` might hold the meaning better.